### PR TITLE
CN-4800 get coins transaction details

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/atoms/TransactionRow/TransactionRow.js [hunk 1]
```diff
@@ -42,11 +43,14 @@ class TransactionRow extends Component {
...
-            <Icon name={iconName} height="16" width="16" fill={color} />
+            <Icon name={iconName} height="16" width="16" fill={color}/>
             <View style={style.amounts}>
               <View>
                 <CelText weight="600" type="H3">
-                  {formatter.usd(transaction.amount_usd)}
+                  {transaction.amount_usd ?
+                    formatter.usd(transaction.amount_usd) :
+                    `${formatter.usd(transaction.fiat_amount, { symbol: "" })}`
+                  } {transaction.fiat_amount && transaction.fiat_currency}
```
Comment (fjovakaric) [inline app/components/atoms/TransactionRow/TransactionRow.js]: move to method `getFiatAmount`, use `formatter.usd(1000, { symbol: 'EUR' })` for displaying fiat amounts

File: app/components/screens/GetCoinsTransactionDetails/GetCoinsTransactionDetails.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        },
+      ]
+    }
+
+    return sortedTransaction
+  }
+
+  getStatusType = status => {
+    switch (status) {
+      case "approved":
```
Comment (fjovakaric) [inline app/components/screens/GetCoinsTransactionDetails/GetCoinsTransactionDetails.js]: use `SIMPLEX_PAYMENT_STATUSES`

File: app/components/screens/TransactionDetails/TransactionDetailsSections.js [hunk 3]
```diff
@@ -43,7 +43,12 @@ export const InfoSection = ({ transaction, transactionProps }) => (
       color={STYLES.COLORS.MEDIUM_GRAY}
       type="H3"
       align="center"
-    >{`${formatter.usd(transaction.amount_usd)} USD`}</CelText>
+    >
+      {transaction.amount_usd ?
+        formatter.usd(transaction.amount_usd) :
+        `${formatter.usd(transaction.fiat_amount, { symbol: "" })}`
+      } {transaction.fiat_amount && transaction.fiat_currency}
```
Comment (fjovakaric) [inline app/components/screens/TransactionDetails/TransactionDetailsSections.js]: use formatter.usd with symbol for fiat

File: app/components/screens/GetCoinsTransactionDetails/GetCoinsTransactionDetails.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+      fiat_currency: transactionParams.fiat_currency,
+      coin: transactionParams.coin,
+    }
+
+
+    const transactionInfoProps = {
+      iconName: "TransactionCC",
+      color: this.getStatusType(transactionParams.status).color,
+      statusText: formatter.capitalize(this.getStatusType(transactionParams.status).text),
+    }
```
Comment (fjovakaric) [inline app/components/screens/GetCoinsTransactionDetails/GetCoinsTransactionDetails.js]: use `...this.getStatusType(status)`
